x-pack/plugins/cloud_security_posture/public/components/csp_inline_description_list.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiDescriptionList, useEuiTheme } from '@elastic/eui';
+import React from 'react';
+import { EuiDescriptionListProps } from '@elastic/eui/src/components/description_list/description_list_types';
+
+const getModifiedTitlesListItems = (listItems: EuiDescriptionListProps['listItems']) =>
+  listItems
+    ?.filter((item) => !!item?.title && !!item?.description)
+    .map((item) => ({ ...item, title: `${item.title}:` }));
+
+// eui size m is 12px which is too small, and next after it is base which is 16px which is too big
+const customFontSize = '1rem';
```

Comment:
I would skip it, but 12 looks too small, and 16 is too big. there are plenty of 14px in kibana including the `add integrations` button which features on every page so this won't be an odd decision.